examples/basic_query/example_client.py

```diff
@@ -34,37 +18,45 @@ def predict(host, uid, x):
     print("'%s', %f ms" % (r.text, latency))
 
 
-def add_mnist_app(host):
+def add_example_app(host):
```

Comment:
To prevent future inconsistencies, it seems like a good idea to import `clipper_manager` and invoke `register_application`. We should do the same with `add_example_model` by calling `clipper_manager.register_external_model`.